app/letters/utils.py

```diff
@@ -61,12 +58,7 @@ def get_bucket_name_and_prefix_for_notification(notification):
         bucket_name = current_app.config['TEST_LETTERS_BUCKET_NAME']
     else:
         bucket_name = current_app.config['LETTERS_PDF_BUCKET_NAME']
-        if notification.sent_at:
-            folder = "{}/".format(notification.sent_at.date())
-        elif notification.updated_at:
-            folder = get_folder_name(notification.updated_at, False)
-        else:
-            folder = get_folder_name(notification.created_at, False)
+        folder = get_folder_name(notification.created_at, False)
```

Comment:
it would be nice to use a keyword param here rather than having a context-less boolean being passed in (so `get_folder_name(notification.created_at, dont_use_sending_date=False)`)

if you want, we can force that in the function definition by including a `*` param before. `def get_folder_name(_now, *, dont_use_sending_date)` will force you to make dont_use_sending_date a keyword param

Reply:
oh I didn't know you could do that - nice